<launch>
  <node pkg="final_challenge_2022" type="track_detect.py" name="track_detect" output="screen">
    <rosparam command="load" file="$(find final_challenge_2022)/track_detect_params.yaml"/>
    <param name="send_debug_image" type="bool" value="true"/>
  </node>
  
  <!-- <include file="$(find zed_wrapper)/launch/zed_no_tf.launch"/> -->
</launch>
